code/modules/spells/aoe_turf/conjure/druidic_spells.dm

```diff
@@ -21,16 +21,14 @@
 	desc = "This spell summons a flock of spooky space bats."
 	feedback = "SB"
 
-	charge_max = 1200 //2 minutes
-	spell_flags = NEEDSCLOTHES
+	charge_max = 300
 	invocation = "Bla'yo daya!"
 	invocation_type = SpI_SHOUT
 	level_max = list(Sp_TOTAL = 3, Sp_SPEED = 3, Sp_POWER = 3)
-	cooldown_min = 600
 
 	range = 1
 
-	summon_amt = 1
+	summon_amt = 4
```

Comment:
If I understand that correctly, thats 16 times the amount as before ?!

Reply:
Bats are really weak, it takes one or two hits to kill them.